company/values.md

```diff
@@ -1,23 +1,24 @@
 # Sourcegraph values
 
-These values are some of the beliefs and principles that help us achieve our [goals](goals/index.md) and [vision](strategy.md#vision).
+These values are the beliefs and principles that help us achieve our [goals](goals/index.md) and [vision](strategy.md#vision).
 
-This list isn't intended to cover everything we care about; instead, it lists the values that we frequently find useful and refer to. We'll keep this list up to date with the frequently used beliefs and principles (adding, editing, and removing entries as needed). Our hope is that this makes this list more accurate and useful than if it were a list of stale, vague, aspirational, or obvious values.
+- [**Be customer driven**](#be-customer-driven): You proactively work on the right things by continuously orienting your goals around delivering value to your customer.
+- [**Work as a team**](#work-as-a-team): You work collaboratively with your peers, cross-functional teammates, and leadership to create shared success, trust, and belonging.
+- [**High agency**](#high-agency): You have the power and the responsibility to improve Sourcegraph as a company and as a product. You deliver regardless of the circumstances.
```

Comment:
What's behind delivering regardless of circumstances?

Reading the High agency section below, I can't really connect the meaning. This could be easily misinterpreted as "leadership" expects you to deliver even if they don't ensure you have the right context, clear goals, a psychologically safe environment and processes that work for you and not against you, etc.

I think it's important to distinguish between responsibility for these things and ultimate accountability — we want teammates to feel responsible, and empowered, to change and contribute to all these things, but ultimately, it's leadership's job to ensure these are taken care of (i.e. they are accountable).

Reply:
This is an attempt to summarize this tweet (in the context of the whole thread): https://twitter.com/shreyas/status/1276956838840295425

> This could be easily misinterpreted as "leadership" expects you to deliver even if they don't ensure you have the right context, clear goals, a psychologically safe environment and processes that work for you and not against you, etc.

We are of course trying to provide the best environment for everyone to be successful (clear goals, right context, psychological safety, good processes, etc.). It is important to acknowledge that we will never be perfect, and in some cases we will fail. We expect every teammate, regardless of position, to be active agents in identifying/communicating when problems exist and collaborating to fix those problems (not idling standing by assuming someone else will fix a problem, or worse, pointing fingers).

I agree this sentence is less than ideal but don't currently have a good suggestion.

Reply:
Addressed in https://github.com/sourcegraph/about/pull/1760